gaia-media-controls.js

```diff
@@ -0,0 +1,291 @@
+;(function(define){'use strict';define(function(require,exports,module){
+
+/**
+ * Dependencies
+ */
+var Component = require('gaia-component');
+var MediaControlsImpl = require('./lib/media-controls-impl');
+
+var MediaControls = Component.register('gaia-media-controls', {
+  /**
+   * 'createdCallback' is called when the element is first created.
+   */
+  created: function() {
+    console.log('creating gaia-media-controls web component...');
```

Comment:
you probably don't want to keep this line in here. Though you could define an off-by-default debug function and convert to using that.